devito/interfaces.py

```diff
@@ -4,11 +4,12 @@
 from tools import aligned
 import os
 from signal import signal, SIGABRT, SIGINT, SIGSEGV, SIGTERM
+from tempfile import gettempdir
 import sys
 import atexit
 
 
-__all__ = ['DenseData', 'TimeData', 'PointData']
+__all__ = ['DenseData', 'TimeData', 'PointData', 'MemmapManager', 'SymbolicData']
```

Comment:
sphinx-apibuild command won't add modules that are not in __all__ to the autodoc directives when auto-generating the project, but since we have already all the configuration files in place there's no need to keep them there. 

The autodoc directives are specified in docs/devito.rst. Adding directives to that files is enough to generate documentation.